src/interfaces/element-props.ts

```diff
@@ -1,9 +1,32 @@
+export const Phases = {
+  Gas: "Gas",
+  Liquid: "Liquid",
+  Solid: "Solid"
+} as const;
+
+export const Categories = {
+  DiatomicNonmetal: "diatomic nonmetal",
+  NobleGas: "noble gas",
+  AlkaliMetal: "alkali metal",
+  AlkalineEarthMetal: "alkaline earth metal",
+  Metalloid: "metalloid",
+  PolyatomicNonmetal: "polyatomic nonmetal",
+  PostTransitionMetal: "post-transition metal",
+  TransitionMetal: "transition metal",
+  Lanthanide: "lanthanide",
+  Actinide: "actinide"
+} as const;
+
+export type Category = typeof Categories[keyof typeof Categories];
+
+export type Phase = typeof Phases[keyof typeof Phases];
```

Comment:
```suggestion
export const Phases = {
  Gas: "gas",
  Liquid: "liquid",
  Solid: "solid"
} as const;
export type Phases = typeof Phases[keyof typeof Phases];


export const Categories = {
  DiatomicNonmetal: "diatomic nonmetal",
  NobleGas: "noble gas",
  AlkaliMetal: "alkali metal",
  AlkalineEarthMetal: "alkaline earth metal",
  Metalloid: "metalloid",
  PolyatomicNonmetal: "polyatomic nonmetal",
  PostTransitionMetal: "post-transition metal",
  TransitionMetal: "transition metal",
  Lanthanide: "lanthanide",
  Actinide: "actinide"
} as const;
export type Categories = typeof Categories[keyof typeof Categories];
```

Reply:
Sir the value for gas , liquid and solid is capatilised for the first letter that's why i have capitalised first letter

Reply:
@maneetgoyal  sir should i change Gas , Liquid and Solid to gas , liquid and solid in the dictionary and the interface as well.

Reply:
No change needed in this case

Reply:
ok @maneetgoyal  sir , please review it one more time if any change is needed .